scipy/spatial/distance.py

```diff
@@ -321,10 +322,35 @@ def directed_hausdorff(u, v, seed=0):
         Input array with M points in N dimensions.
     v : (O,N) array_like
         Input array with O points in N dimensions.
-    seed : int or `numpy.random.Generator` or None, optional
-        Local `numpy.random.RandomState` seed. Default is 0, a random
-        shuffling of u and v that guarantees reproducibility. Also
-        accepts a `numpy.random.Generator` object.
+    rng : int or `numpy.random.Generator` or None, optional
+        Pseudorandom number generator state. Default is 0 so the
```

Comment:
Nit: It'd be nice to have a larger unique number here, for the promotion of general best practices. It doesn't actually _matter_ for this use case (correlations with other uses of `default_rng(0)` don't matter here, unlike with statistical uses of PRNGs), but it's a mental trap for the reader. 0xd19cd893254f53ee is a nice, large arbitrary number I just pulled out of my `/dev/random`.

Reply:
Right, and I typically do that. I wouldn't mind changing it immediately, but I'd rather but sign up for it if it needs a warning cycle.